src/webvr-polyfill.js

```diff
@@ -95,6 +95,18 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
 
   // Provide the VRDisplay object.
   window.VRDisplay = VRDisplay;
+
+  // Provide vrEnabled
+  var that = this;
+  Object.defineProperty(navigator, 'vrEnabled', {
+    get: function () {
+      return that.isCardboardCompatible() &&
```

Comment:
perhaps just have a separate `var` for this boolean outside so you don't have to inherit the entire scope?